dev-tools/releng/src/main.rs

```diff
@@ -477,6 +513,19 @@ async fn main() -> Result<()> {
         }};
     }
 
+    let incorp_version = version.major;
```

Comment:
Wondering if we should make this `MAJOR.0.0.0` just for symmetry with the rolls we've been doing for respins?